src/content/eth2/merge/index.md

```diff
@@ -44,24 +44,27 @@ Plus many more.
 
 [More on Ethereum1.x](/learn/#eth-1x)
 
-These improvements all have a place in Eth2 so it’s likely that their progress may affect the timing of the docking. -->
+These improvements all have a place in Eth2 so it’s likely that their progress may affect the timing of the merge. -->
 
 ## After the merge {#after-the-merge}
 
-This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. At this point Ethereum will have the scale, security and sustainability outlined in its [Eth2 vision](/eth2/vision/).
+This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. 
+!!!!!!! At this point Ethereum will have the scale, security and sustainability outlined in its [Eth2 vision](/eth2/vision/).
 
 ## Relationship between upgrades {#relationship-between-upgrades}
 
 The Eth2 upgrades are all somewhat interrelated. So let’s recap how the merge relates to the other upgrades.
 
 ### The merge and the Beacon Chain {#docking-and-beacon-chain}
 
-Once the merge happens, stakers will be assigned to validate the Ethereum mainnet. Just like with the shard chains. [Mining](/developers/docs/consensus-mechanisms/pow/mining/) will no longer be required so miners will likely invest their earnings into staking in the new proof-of-stake system.
+Once the merge happens, stakers will be assigned to validate the Ethereum mainnet. [Mining](/developers/docs/consensus-mechanisms/pow/mining/) will no longer be required so miners will likely invest their earnings into staking in the new proof-of-stake system.
 
 <ButtonLink to="/eth2/beacon-chain/">The Beacon Chain</ButtonLink>
 
 ### The merge and shard chains {#docking-and-shard-chains}
 
-With mainnet becoming a shard, the successful implementation of the shard chains is critical to this upgrade. It’s likely that the transition will play an important role in helping the community to decide whether to roll out a second upgrade to sharding. This upgrade will make the other shards like mainnet: they’ll be able to handle transactions and smart contracts and not just provide more data.
+Originally, plans were made to implement shard chains prior to the incorporation of the Ethereum mainnet as a primary means of scaling. As research and development of layer 2 scaling solutions have boomed, the priority of core Ethereum devs shifted to expediting the switch a fully proof-of-stake driven mainnet. This means that the remaining shard chains will not be complete prior to switching mainnet to proof-of-stake.
+
+Currently the plan remains for shard chains to be rolled out after mainnet has merged. This will be an ongoing assessment from the community as to the need for potentially multiple rounds of shard chains to allow for endless scalability.
```

Comment:
this second sentence is hard to read. I'm not sure it's even needed on the merge page. Maybe just simplify paragraph to:

"Shard chains will become the focus of the community once mainnet has merged."

Reply:
still outstanding